test/general/buffer_wrapper.pass.cpp

```diff
@@ -34,7 +34,8 @@ struct test_buffer_wrapper
         EXPECT_TRUE(end - begin == size, "wrong effect of iterator's operator - iterator");
 
         auto buf = begin.get_buffer();
-        EXPECT_TRUE(sycl::host_accessor(buf, sycl::read_only).get_pointer() == data, "wrong effect of iterator's method get_buffer");
+        T* actual_data = sycl::host_accessor<uint32_t, 1, sycl::access_mode::read>(buf).get_pointer();

```

Comment:
It's better to use `T` instead of hardcoded `uint32_t`

Reply:
Done